src/D2_model_constraints.py

```diff
@@ -1,2 +1,40 @@
+import pyomo.environ as po
+
+
 def modelling_constraints():
     return
+
+
+def standby_consumption(
+    model, bus_electricity, bus_H2, electrolyser, standby_sink, efficiency
+):
+    """
+    This function forces either the electrolyser or the strandy-by sink consumes 5% of
+    the investment optimization of the electrolyser
+    :param model:
+    energy model
+    :param bus_electricity:
+    bus of electricity
+    :param bus_H2:
+    bus of H2
+    :param electrolyser:
+    electrolyser transformer
+    :param standby_sink:
+    sink for stand-by consumption. It only consumes when not the electrolyser is not producing H2
```

Comment:
Is this true? I mean, you managed to get the "right behaviour" by adding /10 here (somewhat arbitrarily), so this description should say that this may not be right onto the dot.